lib/cff/file.rb

```diff
@@ -65,7 +65,7 @@ def self.read(file)
       content = ::File.read(file)
       comment = File.parse_comment(content)
 
-      new(YAML.safe_load(content, [Date]), comment)
+      new(YAML.safe_load(content, [Date, Time]), comment)
```

Comment:
This was needed for some of the fixtures; not sure if this is the best solution tho 🤔 thoughts?

Reply:
I'm surprised it's needed as I thought CFF only used dates and not full timestamps. Given the discussion over at citation-file-format/citation-file-format#202 we may not need either `Date` or `Time` for version 2.0?

Reply:
Looks good! In case we do, the tests should catch that!